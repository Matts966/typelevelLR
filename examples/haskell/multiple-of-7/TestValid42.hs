
import MultipleOf7

-- 42 = 0b101010

main :: IO ()
main = print $ begin |> o |> i |> o |> i |> o |> i |> o |> end
