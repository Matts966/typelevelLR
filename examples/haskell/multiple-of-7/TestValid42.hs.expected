RuleS0 (Rule01 (Rule10 (Rule21 (Rule50 (Rule31 (Rule00 Accept))))))
